### PR TITLE
Simplify and generate only one pair

### PR DESCRIPTION
**Pull request by tynes**
Simplify the API so that it only generates one cert, that works for our usecase currently
We eventually want to move towards using ACME

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,9 +11,9 @@ RUN apt-get update && apt-get install -y git bash openssl curl ca-certificates \
 
 FROM nginx:1.13
 
-LABEL "com.github.repo"="bpanel-org/bpanel"
+LABEL "com.github.repo"="bpanel-org/securityc"
 LABEL "maintainer"="bcoin"
```
Comment (bucko13) [inline Dockerfile]: should this be `bcoin` or `bpanel`?

File: README.md [hunk 2]
```diff
@@ -20,28 +20,17 @@ for rendering templates with environmental variables.
...
-Generate a CA cert/key pair or provide a CA cert/key pair to create leaf certificate cert/key pairs.
-This is useful if you need TLS termination while you are developing, for example some hardware
+Generate a CA cert/key pair using certstrap or provide a CA cert/key pair to
+create leaf certificate cert/key pairs.
+Optionally bring up `nginx` to use as a reverse proxy which is useful
+if you need TLS termination, for example some hardware
 wallet libraries like [bledger](https://github.com/bcoin-org/bledger) require HTTPS.
-securityc is configured with environmental variables. An arbitrary number of certs can be generated,
-to properly bundle the inputs per cert, the environmental variables must follow the schema:
+securityc is configured with environmental variables.
```
Comment (bucko13) [inline README.md]: probably just `environment variables` right?

File: README.md [hunk 3]
```diff
@@ -20,28 +20,17 @@ for rendering templates with environmental variables.
...
-```
-
-The prefix `SECURITYC` ensures that there are no collisions with other environmental variables.
-The `{APP_NAME}` refers to an application that a cert/private key should be generated for.
-The `{ARG_NAME}` refers to an argument that the script needs to generate the cert/private key pair.
-The script will be invoked once for each `{APP_NAME}` that has all of the appropriate arguments.
-NOTE: valid `{APP_NAME}`s contain `[A-Z]`, please do not include `_` or other characters
-
-The `{ARG_NAME}`s that can be provided are:
+TLS Certificate generation uses these environmental variables:
```
Comment (bucko13) [inline README.md]: Same? (I might be wrong. This is how I feel like I've heard it said though)

**Comment by bucko13** [comment]
I guess the diagram is less applicable while this is in its own repo, but might be good to be clear somewhere in the readme that this is meant to be used as a separate container running in parallel with other containers acting as a proxy. Also how would you use this with the other containers? Do they need to be made aware of this TLS layer?

**Comment by bucko13** [comment]
With those env files now removed in this PR, the implementor will have to make sure to manually set these all themselves (and presumably gitignore as well). Does it make sense to have some kind of base, default, or sample conf that can be imported by default and/or easily used as a reference?

**Comment by tynes** [comment]
I have a config file committed into the root of `bpanel/bpanel` which should be enough I think

**Comment by bucko13** [comment]
Yeah, I saw that. Just wondering if there is an intention for people to be using this in their own separate project, there's nothing clearly that can be used (unless you go digging into another repo)

**Review by bucko13 — APPROVED** [review]
Nice!

**Pull request merged**